crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -105,15 +139,37 @@ pub struct StandardMaterial {
 
     /// Support two-sided lighting by automatically flipping the normals for "back" faces
     /// within the PBR lighting shader.
-    /// Defaults to false.
+    ///
+    /// Defaults to `false`.
     /// This does not automatically configure backface culling, which can be done via
     /// `cull_mode`.
     pub double_sided: bool,
-    /// Whether to cull the "front", "back" or neither side of a mesh
-    /// defaults to `Face::Back`
+
+    /// Whether to cull the "front", "back" or neither side of a mesh.
+    ///
+    /// Defaults to `Face::Back`.
     pub cull_mode: Option<Face>,
+
+    /// Whether to apply only the base colour to this material.
+    ///
+    /// Normals, occlusion textures, roughness, metallic, reflectance and
+    /// emissive are ignored if this is set to `true`.
     pub unlit: bool,
+
+    /// How to interpret the alpha channel of the `base_color_texture`.
+    ///
+    /// By default, it's `Opaque`, therefore completely ignored.
+    /// Note that currently bevy handles poorly semi-transparent textures. You
+    /// are likely to encounter the following bugs:
+    ///
+    /// - When two `AlphaMode::Blend` materials occupy the same pixel, only one
+    ///   material's color will show.
+    /// - If a different mesh is both "in front" and "behind" a non-opaque material,
+    ///   bevy won't know which material to display in front, which might result in
+    ///   flickering.
     pub alpha_mode: AlphaMode,
+
+    /// Re-arange depth of material, useful to avoid z-fighting.
```

Comment:
Needs to note what is closer / farther, and what the default value is.

Reply:
I'm honestly not sure how it works, could @robtfm chime in?

Reply:
a positive bias will cause a material to be rendered as though it is closer to the camera, and a negative bias will cause it to be rendered as though it is further away. 
note this only affects the render order, it does not change the depth of the fragment for stencil testing or for depth writing.
defaults to 0.

Reply:
"only affect render order" means: "only affects overlapping materials" meaning: "only useful to fix z-fighting"?

Reply:
er... for opaque / alpha masked materials it only affects overlapping materials, yes. the render order doesn't affect the final pixel except when the depths are equal, in which case the first rendered material "claims" the spot in the depth buffer and stops everything else from being output.

for transparent materials it can cause a change in the final output for non-overlapping materials if the distance between them is smaller than the bias, by changing which one gets rendered first.